### DISCUSSION TITLE
Integration with RxDB

### DISCUSSION THREAD

**Discussion by endlessuniverse** [refinedev/refine#2057] [Ideas]
https://rxdb.info/replication-graphql.html
Examples using Zustand:
https://github.com/platyplus/platyplus/tree/master/libs/data-access/react-rxdb-hasura
https://stackoverflow.com/questions/68656118/websql-threw-an-error-error-error-code-1-no-such-table-document-store

**Comment by omeraplak** [comment]
Hey @endlessuniverse ,
Thanks so much for contacting us! I think it would be great too! Frankly, I don't have any predictions about when we can add it.
If you try to do it and you need help, we will be happy to help you.

btw, if you haven't been to our [Discord](https://discord.gg/UuU3XCc3J5) yet, we are waiting for you!
